schemas/org.gnome.shell.extensions.github.notifications.gschema.xml

```diff
@@ -5,6 +5,11 @@
       <default>''</default>
       <summary>Your github handle</summary>
     </key>
+    <key name="domain" type="s">
+      <default>'github.com'</default>
+      <summary>Your github url</summary>
+      <description>Set this variable if you are using GitHub Enterprise</description>
```

Comment:
`<summary>Your github domain</summary>`
`<description>Change this domain only if you are using GitHub Enterprise</description>`